.github/workflows/ci.yml

```diff
@@ -5,13 +5,27 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
-    name: Ruby ${{ matrix.ruby }} / ${{ matrix.gemfile }}
+    name: Ruby ${{ matrix.ruby }} / ${{ matrix.gemfile }} / ${{ matrix.database }}
     strategy:
       matrix:
         gemfile: [Gemfile, gemfiles/rails_6_0.gemfile, gemfiles/rails_edge.gemfile]
         ruby: [2.7, 3.0]
+        database: [sqlite, postgres]
     env:
       BUNDLE_GEMFILE: ${{ matrix.gemfile }}
+      DB: ${{ matrix.database }}
+    services:
+      postgres:
```

Comment:
Is there a way to have a service run conditionally? Better yet, the environment that runs CI already has postgres, so we shouldn't need an external service like this:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#databases

Reply:
> Is there a way to have a service run conditionally?

As far as I can tell, there's no way to start a service conditionally. https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices

The github guide (https://docs.github.com/en/actions/guides/creating-postgresql-service-containers) uses docker for postgres, so it shouldn't be that bad. Also, using containers gives us reproducibility and composability which may amortize the cost of spinning up extra service when sqlite testing.